client/src/dataProviders/noticesDataProviders.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import {dispatchReceivedData} from './dataProvidersUtils'
+
+export const noticesProvider = () => ({
+  ref: 'notices',
+  getData: [
+    fetch,
+    `${process.env.REACT_APP_API_URL || ''}/api/o/list_obstaravania`,
```

Comment:
I see this is defined in ```.env.development```. Will this field be present also in production?

Reply:
no need in production, that's what the `|| ''` is there for